components/brave_wallet/browser/brave_wallet_constants.h

```diff
@@ -1489,6 +1489,7 @@ inline constexpr char kBitcoinMainnetRpcEndpoint[] =
     "https://bitcoin-mainnet.wallet.brave.com/";
 inline constexpr char kBitcoinTestnetRpcEndpoint[] =
     "https://blockstream.info/testnet/api/";
+inline constexpr char kMeldRpcEndpoint[] = "https://api.meld.io";
```

Comment:
Why not use a Brave proxy and also avoid adding a build flag for the API key?

Reply:
++ this is a blocker for the privacy review. Let's update this to use a proxy